charts/ingress-nginx/README.md

```diff
@@ -435,20 +435,37 @@ Kubernetes: `>=1.20.0-0`
 | defaultBackend.autoscaling.targetCPUUtilizationPercentage | int | `50` |  |
 | defaultBackend.autoscaling.targetMemoryUtilizationPercentage | int | `50` |  |
 | defaultBackend.containerSecurityContext | object | `{}` | Security Context policies for controller main container. See https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/ for notes on enabling and using sysctls # |
-| defaultBackend.enabled | bool | `false` |  |
+| defaultBackend.defaultBackendMessage | string | `"Default Backend - 404"` |  |
+| defaultBackend.enabled | bool | `true` |  |
 | defaultBackend.existingPsp | string | `""` | Use an existing PSP instead of creating one |
 | defaultBackend.extraArgs | object | `{}` |  |
 | defaultBackend.extraEnvs | list | `[]` | Additional environment variables to set for defaultBackend pods |
-| defaultBackend.extraVolumeMounts | list | `[]` |  |
-| defaultBackend.extraVolumes | list | `[]` |  |
+| defaultBackend.extraVolumeMounts[0].mountPath | string | `"/usr/share/nginx/html/index.html"` |  |
+| defaultBackend.extraVolumeMounts[0].name | string | `"config"` |  |
+| defaultBackend.extraVolumeMounts[0].readOnly | bool | `true` |  |
+| defaultBackend.extraVolumeMounts[0].subPath | string | `"index.html"` |  |
+| defaultBackend.extraVolumeMounts[1].mountPath | string | `"/var/cache/nginx/"` |  |
+| defaultBackend.extraVolumeMounts[1].name | string | `"nginx-empty"` |  |
+| defaultBackend.extraVolumeMounts[2].mountPath | string | `"/var/run/"` |  |
+| defaultBackend.extraVolumeMounts[2].name | string | `"nginx-run"` |  |
+| defaultBackend.extraVolumeMounts[3].mountPath | string | `"/etc/nginx/conf.d/default.conf"` |  |
+| defaultBackend.extraVolumeMounts[3].name | string | `"nginx-conf"` |  |
+| defaultBackend.extraVolumeMounts[3].subPath | string | `"default.conf"` |  |
+| defaultBackend.extraVolumes[0].configMap.name | string | `"default-backend-message-configmap"` |  |
+| defaultBackend.extraVolumes[0].name | string | `"config"` |  |
+| defaultBackend.extraVolumes[1].emptyDir | object | `{}` |  |
+| defaultBackend.extraVolumes[1].name | string | `"nginx-empty"` |  |
+| defaultBackend.extraVolumes[2].emptyDir | object | `{}` |  |
+| defaultBackend.extraVolumes[2].name | string | `"nginx-run"` |  |
+| defaultBackend.extraVolumes[3].configMap.name | string | `"default-backend-nginx-conf-configmap"` |  |
+| defaultBackend.extraVolumes[3].name | string | `"nginx-conf"` |  |
```

Comment:
Probably a better way to do this is to leave it as `default.extraVolumes` and `default.extraVolumeMounts` and set the defaults just in the `values.yaml` file. Filling the slice is a bit too verbose like this, if you could instead put the entire default value for `defaultBackend.extraVolumes` and `defaultBackend.extraVolumeMounts` it'd be better.

Reply:
Ok i will work on it. Good point

Reply:
But I meant changing to default.extraVolumes we want to split (as before) for deployments, I think that is better to do that than use the same for both deployments (ingress-controller & DefaultBackend) or maybe I didn't get your point